crates/bindings-csharp/Runtime/bindings.c

```diff
@@ -36,44 +37,63 @@ OPAQUE_TYPEDEF(ConsoleTimerId, uint32_t);
 #define IMPORT(ret, name, params, args) STDB_EXTERN(name) ret name params;
 #endif
 
-IMPORT(void, console_log,
-       (LogLevel level, const uint8_t* target_ptr, uint32_t target_len,
-        const uint8_t* filename_ptr, uint32_t filename_len, uint32_t line_number,
-        const uint8_t* message_ptr, uint32_t message_len),
-       (level, target_ptr, target_len, filename_ptr, filename_len, line_number,
-        message_ptr, message_len));
-
 IMPORT(Status, table_id_from_name,
        (const uint8_t* name, uint32_t name_len, TableId* id),
        (name, name_len, id));
+IMPORT(Status, index_id_from_name,
+       (const uint8_t* name, uint32_t name_len, IndexId* id),
+       (name, name_len, id));
 IMPORT(Status, datastore_table_row_count,
        (TableId table_id, uint64_t* count),
        (table_id, count));
-IMPORT(Status, datastore_table_scan_bsatn, (TableId table_id, RowIter* iter),
+IMPORT(Status, datastore_table_scan_bsatn,
+       (TableId table_id, RowIter* iter),
        (table_id, iter));
-IMPORT(Status, datastore_insert_bsatn, (TableId table_id, const uint8_t* row_ptr, size_t* row_len_ptr),
-       (table_id, row_ptr, row_len_ptr));
+IMPORT(Status, datastore_btree_scan_bsatn,
+       (IndexId index_id, const uint8_t* prefix, uint32_t prefix_len, ColId prefix_elems,
+        const uint8_t* rstart, uint32_t rstart_len, const uint8_t* rend, uint32_t rend_len, RowIter* iter),
+       (index_id, prefix, prefix_len, prefix_elems, rstart, rstart_len, rend, rend_len, iter));
+IMPORT(Status, iter_by_col_eq,
+       (TableId table_id, ColId col_id, const uint8_t* value,
+        uint32_t value_len, RowIter* iter),
+       (table_id, col_id, value, value_len, iter));
+IMPORT(Status, iter_start_filtered,
+       (TableId table_id, const uint8_t* filter, uint32_t filter_len,
+        RowIter* iter),
+       (table_id, filter, filter_len, iter));

```

Comment:
Also seems to accidentally re-add removed ABIs.

In general, couldn't you just revert this file (almost) entirely? Looks like all the necessary functions were present already, just shuffled around.